.github/workflows/ci.yml

```diff
@@ -4,6 +4,9 @@ on:
   workflow_dispatch:
   push:
   pull_request:
+    types:
+      - opened
+      - reopened
     branches:
       - main
```

Comment:
I think this `pull_request` block is saying, run GHA when opening/reopening PRs against main? Is there a separate flow for PRs against the release branches? I might be misunderstanding

Reply:
> Just want to make sure I understand - the problem that was solved (and is being simplified now) is when a new commit is pushed to an open PR, GHA would run twice (once for the open PR and another for the push). Is that understanding right?

Yes, that is correct.

Reply:
No there isn't a separate flow. Pinning `main` is to ensure we trigger a CI run when there's a PR from a forked repo.
For release branches, we don't create a PR, we create a new branch and might push to it. When we push to it, a PR will be triggered.
Should we add a release branch pattern for pull_request triggers? (in addition to main) @lornasong That would help IF we open PRs to release branches.

Reply:
Addressed in https://github.com/hashicorp/consul-terraform-sync/pull/960/commits/0adbf39fb055f958271851d9ae063d990050e782

Reply:
Thanks for addressing even though it's outside the original PR change